debian/links

```diff
@@ -0,0 +1 @@
+/lib/x86_64-linux-gnu/libnss_envhosts.so /lib/x86_64-linux-gnu/libnss_envhosts.so.2
```

Comment:
Is this necessary? We're not planning to ever link anything to this, so ABI compat is a non-issue.

Reply:
Yep, I can get rid of it.

Reply:
Got rid of it.